### PR TITLE
Fix for bouncing sidecar's gap

### PR DESCRIPTION
**Pull request by justincorrigible**

### CODE REVIEW

File: client/less/map.less [hunk 1]
```diff
@@ -307,4 +306,4 @@
...
-}
+}
```
Comment (BerkeleyTrue) [inline client/less/map.less]: Files should end with new line

File: client/main.js [hunk 2]
```diff
@@ -355,8 +355,11 @@ $(document).ready(function() {
   function showMap() {
     if (!main.isMapAsideLoad) {
       var mapAside = $('<iframe>');
-      mapAside.attr('src', '/map-aside');
-      $('.map-aside').append(mapAside);
+      mapAside.attr({
+        src: '/map-aside',
+        scrolling: 'yes'
+        });
```
Comment (BerkeleyTrue) [inline client/main.js]: This indent should line up with the indent on line 358

**Comment by SaintPeter** [comment]
This fix breaks the Wiki button.

**Comment by SaintPeter** [comment]
:+1:

**Pull request merged**